mitodl/micromasters
# Fixed alter data commands

#### What are the relevant tickets?

More fixes related to #1833 
Rebased on #1833 

#### What's this PR do?

The refactored edX data (#1806) broke our seed data functionality. #1833 fixes up the seed data functionality, and this PR fixes up the alter_data management commands specifically

#### How should this be manually tested?

With seed data loaded, run `./manage.py alter_data --usage` and/or `./manage.py alter_data --help` for usage details. Run through a few of the commands and check the /dashboard page for the Analog Program to make sure the results are what you expect
